### DISCUSSION TITLE
UserIds missing on some traces

### DISCUSSION THREAD

**Discussion by takefy-dev** [langfuse/langfuse#4331] [Support]
### Describe the feature or potential improvement

https://langfuse.com/docs/analytics/daily-metrics-api

I have a composite user id key, i want to be able to filter like this 123456789-*. We then have more possibilities to search

**Comment by marcklingen** [comment]
Thanks for raising this. To make sure I understand this correctly, you'd like to see wildcard filters here instead of exact match?

**Comment by takefy-dev** [comment]
No some users id are empty while I’m sure everything is correctly setup since other user with same request  work

**Reply by marcklingen** [reply]
ok, then let's dive deeper on this in an issue related to userIds being not correctly captured in your case. can you open an issue with details to help reproduce this?

**Comment by takefy-dev** [comment]
I don’t know how to reproduce since it’s always the same i use langgraph and i pass into metadata and langfuse is plug with callbacks methodOn 26 Nov 2024, at 10:41, Marc Klingen ***@***.***> wrote:﻿
ok, then let's dive deeper on this in an issue related to userIds being not correctly captured in your case. can you open an issue with details to help reproduce this?
